src/components/TokenSideBar/TokenSideBarContent.test.tsx

```diff
@@ -1,39 +1,30 @@
 import React from "react";
 import { mount } from "enzyme";
-import TokenSideBar from "./TokenSideBar";
+import TokenSideBarContent from "./TokenSideBarContent";
 
-describe("tokenSideBar", () => {
-  it("should have a Manage Asset heading", () => {
+describe("tokenSideBarContent", () => {
+  // adminAddress === holder
```

Comment:
can you put this comment as a text description? Like "should render is admin address === holder address".